server/index.js

```diff
@@ -1 +1,21 @@
+const express = require('express');
+const app = express();
+app.use(express.json());
+app.use(express.urlencoded());
+const PORT = 3002;
```

Comment:
Owen brought up the great point that instead of hard coding, we do something like 

PORT = process.env.port || 3002

so the environment can pass in a port, but if it isn't supplied then it would fall back to the default.

Reply:
agreed! I thought I changed mine before making this request bc I saw that suggestion, but will for sure update it for the next PR!